tests/suites/test_suite_rsa.function

```diff
@@ -1374,7 +1362,6 @@ void mbedtls_rsa_import_raw( char *input_N,
 
     if( strlen( input_E ) )
         lenE = unhexify( bufE, input_E );
-
```

Comment:
These style changes are to make these tests closer in style to 2.16 and development, right?

Reply:
what style changes?
If you mean the removal of this line, that it was caused when I deleted the `printf` of all the lengths used. Should I revert these line deletions?

Reply:
I don't see any printf in the original diff. Removing the lines adds noise to the diff. It's a bit sloppy if your adding and removing printfs for your own testing purposes results in any diffs.

If the style in 2.16 is not to have newlines at these places in the tests, this is OK.